src/Engine/Manager/PhysicsManager.cpp

```diff
@@ -87,20 +94,34 @@ void PhysicsManager::Update(float deltaTime) {
     }
 }
 
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') The function 'MakeTrigger' is never used. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](http://localhost:9000/coding_rules#rule_key=cppcheck%3AunusedFunction)